packages/fether-electron/src/main/app/constants/index.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+const TRUSTED_HOSTS = ['api.github.com'];
+
+// Disallows users from using Fether with a remote node.
+// SSH tunnels are still possible.
+//
+// References: See trustworthiness of 127.0.0.1 vs localhost: https://letsencrypt.org/docs/certificates-for-localhost/
+const TRUSTED_URLS = [
+  // Allow open 127.0.0.1 using http in dev mode
+  'http://127.0.0.1:3000',
+  'ws://127.0.0.1:8546',
```

Comment:
We should remove the `--ws-interface` and `--ws-port` cli options then (or alternatively dynamically add the CLI interface/port to the trusted_urls)

Reply:
Perhaps we should just remove `--ws-interface` cli option and only allow `127.0.0.1` since it's considered a loopback address (see https://letsencrypt.org/docs/certificates-for-localhost/)

But leave the `--ws-port` as a cli option and add it dynamically as you suggested to `TRUSTED_URLS`.

What do you think?

Reply:
no strong opinion on this. simplest might be to just keep the two cli flags and add the interface:port to the trusted url. but from a security standpoint it'd be pretty bad if somebody manages to launch Fether on sb else's computer with a malicious remote `--ws-interface` (since the account password is sent over rpc for some methods, e.g. account backup --- but then again acct password ≠ private key). only allowing to set a `--ws-port` reduces the attack surface, although then again the attacker might be able to set up a tunnel locally (but if they can do that they can also spawn a malicious process/tunnel listening on :8546 before Fether is launched, and then Fether would connect to it..)

I think the option you suggested is good enough

Reply:
Can we change to ` --ws-origins=parity://1.ui.parity"` instead of `--ws-origins all`?

Reply:
i'm going to have to put off addressing this until the end of the week, as i need to start preparing for a presentation on Thurs

Reply:
@axelchalon i've pushed commits addressing this review comment